src/Docnet.Tests.Integration/PageReaderTests.cs

```diff
@@ -231,6 +232,22 @@ public void GetPageWidthOrHeight_WhenCalledWithScalingFactor_ShouldMach(Input ty
             });
         }
 
+        [Fact]
+        public void GetImage_WhenCalledWithRenderAnnotationsFlag_ShouldRenderAnnotation()

```

Comment:
can we also add a test where multiple flags are used at once?

Reply:
I added a test when the RenderAnnotations and Grayscale flags are both set.  I also added a baseline test to verify that annotations pdf renders as expected when no flags are set.